src/App.jsx

```diff
@@ -23,41 +21,23 @@ function App() {
   }
 
   useEffect(() => {
-    let interval = ''
-    if (idInstance !== '' && apiToken !== '') {
-      interval = setInterval(
-        () =>
-          fetch(
-            `${API_URL}/waInstance${idInstance}/ReceiveNotification/${apiToken}`
-          )
-            .then((resp) => resp.json())
-            .then((data) => {
-              if (data.body.messageData.typeMessage === 'textMessage') {
-                setContacts([...contacts, data.body.senderData.chatName])
-                setMessage(data.body.messageData.textMessageData.textMessage)
-              } else {
-                setContacts([...contacts, data.body.senderData.chatName])
-                setMessage(data.body.messageData.typeMessage)
-              }
-              return data
-            })
-            .then((data) => {
-              fetch(
-                `${API_URL}/waInstance${idInstance}/deleteNotification/${apiToken}/${data.receiptId}`,
-                {
-                  method: 'DELETE',
-                }
-              )
-            })
-            .catch((e) => {
-              console.log('error', e.message)
-            }),
-        5000
-      )
-    }
-    return () => {
-      clearInterval(interval)
-    }
+    if (idInstance === '' || apiToken === '') return
+    ;(async () => {
+
+      const restAPI = whatsAppClient.restAPI({
+        idInstance,
+        apiTokenInstance: apiToken,
+      })
+
+      try {
+        await restAPI.webhookService.startReceivingNotifications()
+        restAPI.webhookService.onReceivingMessageText((body) => {
+          setMessage(body.messageData.textMessageData.textMessage)
+        })
+      } catch (ex) {
```

Comment:
сокращать навзания переменных не над
```suggestion
      } catch (error) {
```